assets/scripts/embedding_tests_dev_0.0.1.min.html

```diff
@@ -6,36 +6,68 @@ <h2>v0.0.1 (minified)</h2>
   the Oppia iframe to the containing page.
 </p>
 
+<style type="text/css">
+  .embedding-content{
+    display: none;
+  }
+</style>
+
 <script src="/assets/scripts/oppia-player-0.0.1.js"></script>
 <script src="/assets/scripts/embedding_tests.js"></script>
+<script src="../../third_party/static/jquery-3.2.1/jquery.min.js"></script>
+
+<input id="embedding-exploration-id" type="text" placeholder="explorationId">
+<input id="embedding-submit-btn" type="submit" value="Update Exploration ID" onclick="onChangeExpId()">
 
-<div>
+<div class="embedding-content">
```

Comment:
Create a parent div around all of this, and give it the name .test-results perhaps (the point of the div is that it changes when you press the submit button, so it's sort of the result of clicking submit).

Reply:
have wrap them all in a div.